### ISSUE TITLE
packages.confluent.io error network

### ISSUE THREAD

**Issue by hishamag-student** [mostafaghadimi/debezium_avro_integration#18]
Hello, please can you resolve with me this issue, it still shown this message error on the console after docker build . 
 => => # curl: (6) Could not resolve host: packages.confluent.io    #   #    #  #                                                                    
 and nothing shown in my jar_files folder, so I can't use plugins in my project.

**Comment by mostafaghadimi** [comment]
Hi @hishamag-student,

It seems that you have trouble downloading the packages from Confluent. Can you use proxy in order to download the artifacts?

**Comment by hishamag-student** [comment]
no, but from ubuntu terminal, when I need to use curl for example I need to use it with sudo.

**Comment by hishamag-student** [comment]
I resolve this adding nameserver 8.8.8.8 and this nameserver 8.8.4.4  to  /etc/resolv.conf
